### DISCUSSION TITLE
Outlook Calendar

### DISCUSSION THREAD

**Discussion by vargeloglu** [community/community#33167] [Other Feature Feedback, Questions, & Ideas]
Hello,

My company uses on-premise exchange server and we would like to display Outlook calendars on our company's employee portal.
What about displaying each individual's calendar on our site for them to see upon login?
In this scenario the employee would log in and see their own calendar events from outlook displaying and editing on our site.
Is that possible?

Note: This site is logged in with the domain account in the active directory.

Thank you.

**Comment by jfullstackdev** [comment]
I'm not very sure of your situation, but if you are asking a Calendar service that can be configured, I think it's same idea as Google Calendar

you can have a Calendar for your work schedules that can be seen by others on the same domain or a private one just for you only

**Comment by vargeloglu** [comment]
Hello,

MS Exchange Server calendar is used. I'm trying to make everyone see and edit their own calendar on the company's web portal.

Can we pull the calendar item to our own web portal via Exchange Web Services?

Thank you.
